app/helpers/hosts_helper.rb

```diff
@@ -251,7 +251,7 @@ def name_field(host)
   end
 
   def build_duration(host)
-    return _("N/A") if host.initiated_at.nil? && host.installed_at.nil?
+    return _('N/A') if host.initiated_at.nil? || host.installed_at.nil?
     if host.installed_at.nil?
       time_ago_in_words(host.initiated_at, include_seconds: true) + " (in progress)"
```

Comment:
This code is now unreachable?

Reply:
Looks unreachable, yes.

Reply:
Bite me, but I do see a path: when `host.installed_at.nil?` is true?!? Explain please.

Reply:
@lzap Function will return on line 254 if host.installed_at.nil? is true.

Reply:
I see.

Reply:
I will take a look next week. My GH backlog is not getting slimmer.

Reply:
Solved in https://github.com/theforeman/foreman/pull/6603